subprojects/common/okhttp/src/main/kotlin/com/avito/http/RequestBodies.kt

```diff
@@ -34,6 +34,7 @@ fun isPlaintext(buffer: Buffer): Boolean {
         val prefix = Buffer()
         val byteCount = if (buffer.size < 64) buffer.size else 64
         buffer.copyTo(prefix, 0, byteCount)
+        @Suppress("UnusedPrivateMember")
```

Comment:
Didn't get the problem with that code

Reply:
`i` is unused

Reply:
Could we change `i` to `_` and remove `Suppress`?

Reply:
nope, 
![Uploading image.png…]()

Reply:
nope, 
![image](https://user-images.githubusercontent.com/1105133/99606325-90b17f80-2a1a-11eb-90f6-ec02f95baf0e.png)

Reply:
It shows that we don't need an index here at all:

```kotlin
repeat(15) { ... }
```